Compilation error on Windows 11: '__cpuid' was not declared in this scope

While trying to compile the Gismo library on Windows 11 using Clion with MinGW compiler, i got the following compilation error:
```
FAILED: src/CMakeFiles/gsCore.dir/gsCore/gsSysInfo.cpp.obj
C:\PROGRA~1\JETBRA~1\CLION2~1.2\bin\mingw\bin\G__~1.EXE -Dgismo_EXPORTS -IC:/Users/albeb/Downloads/gismo-stable1/gismo-stable/src -IC:/Users/albeb/Downloads/gismo-stable1/gismo-stable/external -IC:/Users/albeb/Downloads/gismo-stable1/gismo-stable/cmake-build-release -IC:/Users/albeb/Downloads/gismo-stable1/gismo-stable/optional -Wall -Wno-long-long -Wunused-variable -Wa,-mbig-obj -Wno-unknown-pragmas -march=core2 -mmmx -msse2 -msse3 -msse -mssse3 -mno-avx -mno-avx2 -mno-fma -mno-sse4.1 -mno-sse4.2 -mno-sse4a -mno-avxvnni -mno-avx5124fmaps -mno-avx5124vnniw -mno-avx512bf16 -mno-avx512bitalg -mno-avx512bw -mno-avx512cd -mno-avx512dq -mno-avx512er -mno-avx512f -mno-avx512fp16 -mno-avx512ifma -mno-avx512pf -mno-avx512vbmi2 -mno-avx512vbmi -mno-avx512vl -mno-avx512vnni -mno-avx512vp2intersect -mno-avx512vpopcntdq -mno-amx-bf16 -mno-amx-int8 -mno-amx-tile -mno-adx -mno-aes -mno-bmi2 -mno-enqcmd -mno-f16c -mno-fsgsbase -mno-fxsr -mno-gfni -mno-hreset -mno-lzcnt -mno-movbe -mno-movdir64b -mno-movdiri -mno-mpx -mno-pclmul -mno-pconfig -mno-pku -mno-popcnt -mno-prfchw -mno-prefetchwt1 -mno-ptwrite -mno-rdpid -mno-rdrnd -mno-rdseed -mno-rtm -mno-serialize -mno-sha -mno-tsxldtrk -mno-uintr -mno-vaes -mno-vpclmulqdq -mno-waitpkg -mno-wbnoinvd -mno-xsavec -mno-xsaveopt -mno-xsaves -mno-abm -mno-bmi -mno-cldemote -mno-clflushopt -mno-clwb -O3 -DNDEBUG -std=c++14 -fvisibility=hidden -fno-keep-inline-dllexport -fdiagnostics-color=always -MD -MT src/CMakeFiles/gsCore.dir/gsCore/gsSysInfo.cpp.obj -MF src\CMakeFiles\gsCore.dir\gsCore\gsSysInfo.cpp.obj.d -o src/CMakeFiles/gsCore.dir/gsCore/gsSysInfo.cpp.obj -c C:/Users/albeb/Downloads/gismo-stable1/gismo-stable/src/gsCore/gsSysInfo.cpp
C:/Users/albeb/Downloads/gismo-stable1/gismo-stable/src/gsCore/gsSysInfo.cpp: In static member function 'static std::string gismo::gsSysInfo::getCpuInfo()':
C:/Users/albeb/Downloads/gismo-stable1/gismo-stable/src/gsCore/gsSysInfo.cpp:598:5: error: '__cpuid' was not declared in this scope; did you mean '_clui'?
  598 |     __cpuid(CPUInfo, 0x80000000);
      |     ^~~~~~~
      |     _clui

```
I solved this problem by including the `intrin.h` package after line 20 in the `gsSysInfo.cpp` file.

--- a/src/gsCore/gsSysInfo.cpp
+++ b/src/gsCore/gsSysInfo.cpp
@@ -18,6 +18,9 @@
 
 #if defined(_WIN32) || defined(_WIN64)
 #   include <windows.h>
+#ifdef __MINGW32__
+#   include <intrin.h>
+#endif
 #elif __APPLE__
 #   include <sys/utsname.h>
 #   include <sys/sysctl.h>

